phpstan.neon

```diff
@@ -1,3 +1,7 @@
 parameters:
 	checkMissingIterableValueType: false
 	checkGenericClassInNonGenericObjectType: false
+	ignoreErrors:
+		-
+			message: '#does not accept array<string, array\|string>#'
+			path: %currentWorkingDirectory%/src/HttpRequest.php
```

Comment:
This is fixed in https://github.com/phpstan/phpstan/issues/6927
so we can remove it after update phpstan.